api/ereqs_admin/templatetags/nav_menu.py

```diff
@@ -32,23 +33,30 @@ def append_child(self, child):
         return self._replace(children=self.children + (child,))
 
 
-menu_archetype = MenuItem.new('', children=(
-    MenuItem.new('Policies', children=(
-        MenuItem.new('View all policies',
-                     url=reverse('admin:reqs_policy_changelist')),
-        MenuItem.new('Add new', url=reverse('admin:reqs_policy_add')),
-    )),
-    MenuItem.new('Users', url=reverse('admin:auth_user_changelist')),
-    # Settings depends on the user id, so is added later
-))
+@lru_cache()
+def get_menu_archetype():
```

Comment:
It does feel a bit silly to create an LRU cache for a function with no arguments, but um, it seems to get the job done? 🤷‍♂️